src/__test__/components/VideoCard.test.js

```diff
@@ -0,0 +1,16 @@
+import { getByText } from '@testing-library/dom'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import VideoCard from '../../components/VideoCard'
+import videosMock from '../../youtube-videos-mock.json'
+
+describe('VideoCard testing', () => {
```

Comment:
Maybe in this test file you can check different things on your video card.
Try grabing one item and verifying all the content of the videocard, for example:
- That when rendering the title it matches with the real information you are sending by props.
- That the src of the thumbnail matches with the correct item.
etc, etc.